src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/utils/TransactionUtils.java

```diff
@@ -31,23 +32,40 @@ public boolean doesTransactionHaveALimitedPartnershipSubmission(Transaction tran
     }
 
     public boolean isTransactionLinkedToLimitedPartnershipIncorporation(Transaction transaction, String limitedPartnershipIncorporationSelfLink) {
-        return doChecks(transaction, limitedPartnershipIncorporationSelfLink, FILING_KIND_REGISTRATION);
+        return doIncorporationChecks(transaction, limitedPartnershipIncorporationSelfLink);
+    }
+
+    public boolean isTransactionLinkedToPartnerSubmission(Transaction transaction, String partnerSubmissionSelfLink, String kind) {
```

Comment:
can't we just call the method isTransactionLinkedToPartner

Reply:
I think so. I didn't actually write this code, just moved it. Happy to change this. @mattch1?

Reply:
ah ok leave for now then